instrumentation/lettuce-4.3/build.gradle

```diff
@@ -0,0 +1,24 @@
+dependencies {
+    implementation(project(":newrelic-weaver-api"))
+    implementation(project(":agent-bridge"))
+    implementation 'biz.paluch.redis:lettuce:4.4.1.Final'
+    testImplementation('org.testcontainers:testcontainers:1.17.1')
+    testImplementation('junit:junit:4.13.1')
+
+}
+
+jar {
+    manifest {
+        attributes 'Implementation-Title': 'com.newrelic.instrumentation.lettuce-4.3'
+    }
+}
+
+verifyInstrumentation {
+    passes 'biz.paluch.redis:lettuce:[4.3.0,4.4.4.Final]'
```

Comment:
It's a best practice to use `passesOnly` when you can as it will not only verify the supported version defined in the range but also verify that all versions outside of the range don't verify. Doing this helps to prevent scenarios where multiple instrumentation modules apply and cause double instrumentation.

Also, has the verifier actually been run against this?

Reply:
true, need to do that still...